### PR TITLE
Improvement on the footsteps particle player storing and fixed typo in ConfigPath.java

### PR DESCRIPTION
**Pull request by KKiiya**
Changed hasmap to a list so it will be lighter

### CODE REVIEW

File: bedwars-plugin/src/main/java/com/tomkeuper/bedwars/listeners/InvisibilityPotionListener.java [hunk 1]
```diff
@@ -41,6 +50,47 @@
  * potion or when the potion is gone. It is required because it is related to scoreboards.
  */
 public class InvisibilityPotionListener implements Listener {
+    private final List<Player> invisiblePlayers = new ArrayList<>();
+    private final boolean footstepsEnabled = BedWars.getAPI().getConfigs().getMainConfig().getBoolean(ConfigPath.GENERAL_CONFIGURATION_ENABLE_FOOTSTEPS_ON_INVISIBILITY);
```
Comment (Xxyuri2005xX) [inline bedwars-plugin/src/main/java/com/tomkeuper/bedwars/listeners/InvisibilityPotionListener.java]: maybe you can use the method BedWars#config

Comment (Xxyuri2005xX) [reply bedwars-plugin/src/main/java/com/tomkeuper/bedwars/listeners/InvisibilityPotionListener.java]: BedWars.config.getBoolean(ConfigPath.GENERAL_CONFIGURATION_ENABLE_FOOTSTEPS_ON_INVISIBILITY)

Comment (KKiiya) [reply bedwars-plugin/src/main/java/com/tomkeuper/bedwars/listeners/InvisibilityPotionListener.java]: Oh i didnt know there was that, thanks, ill change it rn

**Review by tomkeuper — APPROVED** [review]

**Comment by tomkeuper** [comment]
As this is only supported up to 1.12 I've added a version check that will just return out of the function when its used on higher versions. Also an issue has been created for anyone willing to try and figure out how to implement this on higher versions (if possible)

Note to server owners: When a player joins with a client above 1.12 they are NOT able to see the particles

**Pull request merged**